test/serde/msgpack/test_msgpack_serde_full.py

```diff
@@ -70,7 +70,7 @@
 ] = make_fixedprecisiontensor
 samples[syft.frameworks.torch.tensors.interpreters.private.PrivateTensor] = make_privatetensor
 
-samples[syft.frameworks.crypten.model.OnnxModel] = make_onnxmodel
+# samples[syft.frameworks.crypten.model.OnnxModel] = make_onnxmodel
```

Comment:
MAYBE: uncomment this

Reply:
yup,i commented while local testing forgot to remove it